js/webpack.config.js

```diff
@@ -131,6 +132,16 @@ module.exports = {
         ]
       }),
       new HappyPack({
+        id: 'less',
```

Comment:
Is there anything that less would give us (for our usage), that PostCSS does not?

Look above at your less file, stock-standard CSS. (Well, you did have the & nested, which is indeed something I do miss from Less & Sass.) However in PostCss - https://github.com/postcss/postcss-nested